lib/install/management/store_files.go

```diff
@@ -189,3 +191,27 @@ func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfi
 	}
 	return nil
 }
+
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) {
+	if d.force {
+		for label, url := range conf.VolumeLocations {
+			// separate the host (datastore name) from the path in the provided path URL
+			pathComponents := strings.SplitN(url.Path, " ", 2)
+			if len(pathComponents) != 2 {
+				log.Warnf("Didn't receive an expected volume store path format: %s", url.Path)
+			}
+
+			log.Infof("Deleting volume store %s on Datastore %s at path %s", label, pathComponents[0], pathComponents[1])
+			if _, err := d.deleteDatastoreFiles(d.session.Datastore, pathComponents[1], d.force); err != nil {
```

Comment:
Oh, you're right. I need to use finder on each datacenter from the paths in conf.. good catch Faiyaz